lib/src/toggle-group/ToggleGroup.tsx

```diff
@@ -35,7 +35,7 @@ const DxcToggleGroup = ({
         setSelectedValue(newSelectedOptions);
       } else setSelectedValue(selectedOption === selectedValue ? null : selectedOption);
     } else if (multiple) {
-      newSelectedOptions = value.map((v) => v);
+      newSelectedOptions = Array.isArray(value) ? value.map((v) => v) : value;
```

Comment:
If it is multiple, `value` should be an array, what's the purpose of iterating throw it to just not change anything (`map(v => v)`)? I would just put `newSelectedOptions = value` and not check anything.

Reply:
I think we can leave this change for a future revision of the component.

Reply:
Perfect! I agree.